packages/ui-docs-plugin/src/index.js

```diff
@@ -36,32 +36,26 @@ DocsPlugin.prototype.apply = function (compiler) {
   const pluginOptions = this.options || {}
   const options = getOptions(pluginOptions)
 
-  compiler.apply(
-    new SingleEntryPlugin(
-      this.context,
-      `!!${require.resolve('./loaders/docs-loader')}?${JSON.stringify(
-        pluginOptions
-      )}!`,
-      'ui-docs'
-    )
-  )
-
-  compiler.apply(
-    new HtmlWebpackPlugin({
-      title: options.title,
-      filename: 'index.html',
-      template: options.template,
-      inject: 'body',
-      chunksSortMode: 'dependency'
-    })
-  )
-
-  compiler.apply(
-    new ScriptExtHtmlWebpackPlugin({
-      defaultAttribute: 'sync',
-      defer: ['ui-docs'] // to ensure that globals loads first
-    })
-  )
+  new SingleEntryPlugin(
+    this.context,
+    `!!${require.resolve('./loaders/docs-loader')}?${JSON.stringify(
+      pluginOptions
+    )}!`,
+    'ui-docs'
+  ).apply(compiler)
```

Comment:
This is just updating a deprecated syntax, that would throw an error if we upgrade to Webpack 5